src/chinks/VerticalAddition/VerticalAddition.java

```diff
@@ -0,0 +1,97 @@
+
+package chinks.VerticalAddition;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class VerticalAddition {
+   public class Node{
+        int data;
+        Node left;
+        Node right;
+        
+        public Node (int a)
+        {
```

Comment:
follow Egyptian braces instead of braces in newline

Egyptian braces looks like:
public Node(){
}

Not
public Node()
{

}

fix this everywhere